tests/pipelines/controlnet/test_controlnet_sdxl.py

```diff
@@ -817,3 +818,162 @@ def test_depth(self):
         original_image = images[0, -3:, -3:, -1].flatten()
         expected_image = np.array([0.4399, 0.5112, 0.5478, 0.4314, 0.472, 0.4823, 0.4647, 0.4957, 0.4853])
         assert np.allclose(original_image, expected_image, atol=1e-04)
+
+
+class StableDiffusionSSD1BControlNetPipelineFastTests(StableDiffusionXLControlNetPipelineFastTests):
```

Comment:
Wondering if this should go into a separate testing script (potentially named `test_controlnet_ssd.py`)? WDYT @patrickvonplaten?

Reply:
Think it's ok to leave here